File: classes/ground.py
from settings import *

class Ground:
    VELOCITY = 5
    IMAGE = GROUND_IMAGE
    WIDTH = GROUND_IMAGE.get_width()

    def __init__(self, y):
        self.y = y
        self.x_0 = 0
        self.x_1 = self.WIDTH

    def move(self):
        self.x_0 -= self.VELOCITY
        self.x_1 -= self.VELOCITY

        if self.x_0 + self.WIDTH < 0:
            self.x_0 = self.x_1 + self.WIDTH
        if self.x_1 + self.WIDTH < 0:
            self.x_1 = self.x_0 + self.WIDTH

    def draw(self, screen):
        screen.blit(self.IMAGE, (self.x_0, self.y))
        screen.blit(self.IMAGE, (self.x_1, self.y))

File: src/Ground.java
import javax.imageio.ImageIO;
import java.awt.*;
import java.awt.image.BufferedImage;
import java.io.BufferedInputStream;
import java.io.IOException;
import java.util.Objects;

class Ground {
    int velocity = 5;
    int width;
    BufferedImage groundImage;
    GamePanel gp;
    int x0 = 0;
    int x1;
    int y;

    Ground(GamePanel gp) throws IOException {
        this.gp = gp;
        groundImage = ImageIO.read(new BufferedInputStream(
                Objects.requireNonNull(getClass().getResourceAsStream("/images/ground.png")
        )));
        width = groundImage.getWidth();
        x1 = width;
        y = gp.height - groundImage.getHeight();
    }

    void draw(Graphics2D g2d) {
        g2d.drawImage(groundImage, x0, y, null);
        g2d.drawImage(groundImage, x1, y, null);
    }

    void update() {
        x0 -= velocity;
        x1 -= velocity;

        if (x0 + width < 0) {
            x0 = x1 + width;
        }
        if (x1 + width < 0) {
            x1 = x0 + width;
        }
    }
}
